src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutPersisterReceiver.cs

```diff
@@ -19,6 +19,16 @@ class TimeoutPersisterReceiver : IDisposable
         public CriticalError CriticalError { get; set; }
         public Address DispatcherAddress { get; set; }
         public TimeSpan TimeToWaitBeforeTriggeringCriticalError { get; set; }
+        internal DateTime NextRetrieval { get; private set; } = DateTime.UtcNow;
+
+        public TimeoutPersisterReceiver()
+        {            
+        }
+
+        public TimeoutPersisterReceiver(Func<DateTime> currentTimeProvider)
```

Comment:
@Particular/nservicebus-maintainers this seems to have broken StructureMap 

https://gitter.im/Particular/NServiceBus?at=59bbaa71c101bc4e3acd1912

@Particular/container-maintainers @SimonCropp @janovesk FYI